src/build/github.rs

```diff
@@ -60,14 +61,18 @@ impl Collector {
     /// Collect contributions (commits, issues, prs) from GitHub for each of
     /// the repositories in the GitHub organizations provided.
     #[instrument(skip(self))]
-    pub(crate) async fn collect_contributions(&self, orgs: &[String]) -> Result<()> {
+    pub(crate) async fn collect_contributions(&self, settings: &Settings) -> Result<()> {
         debug!("collecting contributions");
 
         // Fetch organizations' repositories
         let mut repositories = vec![];
-        for org in orgs {
+        for org in settings.organizations.iter() {
             repositories.extend(self.list_repositories(org).await?);
         }
+        for repo in settings.repositories.iter() {
+            let pair = repo.splitn(2, '/').collect::<Vec<&str>>();
```

Comment:
Could you validate that the format used in the repositories provided is what we expect (`owner/repo`)?

(i.e. the tool will crash if something like `repo` is provided, or will produce some errors if the format used is `https://github.com/owner/repo`).

We could raise a warning when an invalid entry is detected displaying the correct format.

Reply:
Updated to best effort verify the input.